### PR TITLE
Tag JuMPeR v0.5.0

### PR DESCRIPTION
**Pull request by yeesian**
@IainNZ

### CODE REVIEW

File: JuMPeR/versions/0.5.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+JuMP 0.14 0.14.1
```
Comment (tkelman) [inline JuMPeR/versions/0.5.0/requires]: that's an awfully specific range, what's going on there?

Comment (yeesian) [inline JuMPeR/versions/0.5.0/requires]: Was at Iain's request to provide an upperbound, since 0.14.1 is the latest.

Comment (yeesian) [inline JuMPeR/versions/0.5.0/requires]: Oh my bad. Should I bump the require to 0.14.2, and tag a new patch?

Comment (tkelman) [inline JuMPeR/versions/0.5.0/requires]: if the only thing you want to change is the bound, you can just change this file in metadata, leave the tag alone. just be sure to make the same require change on master too

Comment (yeesian) [inline JuMPeR/versions/0.5.0/requires]: Thanks! That's helpful to know

**Comment by yeesian** [comment]
How do I change https://github.com/IainNZ/JuMPeR.jl/releases/tag/v0.5.0 to point to the latest master?

File: JuMPeR/versions/0.5.0/requires (hunk 1 above)
Comment (tkelman) [inline JuMPeR/versions/0.5.0/requires]: will you also want to hold back to 0.14.1, or is the intent to stay on any 0.14.x release?

**Comment by IainNZ** [comment]
Sorry I should have checked this closer, it should be JuMP 0.15

File: JuMPeR/versions/0.5.0/sha1 [hunk 2]
```diff
@@ -0,0 +1 @@
+9c6a3e840d9b74aa8efd6c709f4524466173d2f2
```
Comment (tkelman) [inline JuMPeR/versions/0.5.0/sha1]: this doesn't match the git tag https://github.com/IainNZ/JuMPeR.jl/tree/v0.5.0

Comment (tkelman) [inline JuMPeR/versions/0.5.0/sha1]: Can you rephrase the question?

Comment (yeesian) [inline JuMPeR/versions/0.5.0/sha1]: Is it okay for me to just replace this line with `e161a36abfd541926db9d7591eb6e3ae6ec9bdc4` (based on [the log](https://travis-ci.org/JuliaLang/METADATA.jl/builds/167983919#L234))?

File: JuMPeR/versions/0.5.0/requires [hunk 3]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+JuMP 0.14 0.14.2
```
Comment (tkelman) [inline JuMPeR/versions/0.5.0/requires]: you still want this to be 0.15 though don't you?

Comment (yeesian) [inline JuMPeR/versions/0.5.0/requires]: ugh, i rebased it to some wrong version, sorry!
